Sprint-1/explore/objects.md

```diff
@@ -4,13 +4,17 @@ In this activity, we'll explore some additional concepts that you'll encounter i
 
 Open the Chrome devtools Console, type in `console.log` and then hit enter
 
-What output do you get?
+What output do you get? Answer : ƒ log() { [native code] }
 
-Now enter just `console` in the Console, what output do you get back?
+Now enter just `console` in the Console, what output do you get back? Answer: {debug: ƒ, error: ƒ, info: ƒ, log: ƒ, warn: ƒ, …}
 
 Try also entering `typeof console`
 
 Answer the following questions:
 
-What does `console` store?
+What does `console` store? Answer: console is not designed to store any data. it output with messages , error and other useful data
```

Comment:
The value stored in the variable `console` is an **object**. What can be found inside an object? 
What does the output `{debug: ƒ, error: ƒ, info: ƒ, log: ƒ, warn: ƒ, …}` tells you about the object stored in variable `console`?

Reply:
The {debug: ƒ, error: ƒ, info: ƒ, log: ƒ, warn: ƒ, …} output provides a quick overview of some core methods available on the console object. Each method (like ƒ for function) allows for logging different types of messages, helping with debugging, information display, and error reporting directly within the browser's console.